api/turing/middleware/authorization.go

```diff
@@ -65,6 +66,32 @@ func (a *Authorizer) Middleware(next http.Handler) http.Handler {
 	})
 }
 
+func (a *Authorizer) FilterAuthorizedProjects(
+	user string,
+	projects []mlp.Project,
+	action string,
+) ([]mlp.Project, error) {
+	projectIDs := make([]string, 0)
+	var allowedProjects []mlp.Project
```

Comment:
Nitpick: I think moving this variable declaration just before using it improves readability. i.e moving this line to above 88.

Reply:
agreed